sos/report/plugins/openstack_ceilometer.py

```diff
@@ -26,11 +26,11 @@ def setup(self):
         # Ceilometer
         if self.get_option("all_logs"):
             self.add_copy_spec([
-                "/var/log/ceilometer/*",
+                "/var/log/containers/ceilometer/*",
             ])
         else:
             self.add_copy_spec([
-                "/var/log/ceilometer/*.log",
```

Comment:
.. and also here. When `OpenStack Ceilometer` switched to *only* containerised deployment? What is the latest OSP version that logs into `/var/log/ceilometer/`?

Reply:
The last OSP version logging to /var/log/ceilometer was OSP10, which is out of support since December 15, 2021, see https://access.redhat.com/support/policy/updates/openstack/platform/non-current

Reply:
As these changes are changing the base plugin, we need to consider the non-RH deployments that may consume this change.

Do we have an indication on upstream's stock configuration, or if there is (sizeable) usage of the older version(s)? @ddstreet @slashdd any ideas on Ubuntu based deployments for this?

Reply:
I should probably add the changed path below to the Red Hat section. The path mentioned there does not exist anymore.

Reply:
Yeah this won't work with Ubuntu Openstack deployments that are not using /var/log/containers. @mrunge would it be possible to move all these changes under the RedHat* implementation class so that it does not impact other distros?